framework/behaviors/BlameableBehavior.php

```diff
@@ -97,7 +106,12 @@ public function init()
     protected function getValue($event)
     {
         if ($this->value === null && Yii::$app->has('user')) {
-            return Yii::$app->get('user')->id;
+            $userId = Yii::$app->get('user')->id;
+            if (is_null($userId) && $this->useDefaultValue) {
```

Comment:
`is_null($userId)` => `$userId !== null`

Reply:
@rob006 http://php.net/manual/ru/function.is-null.php#87355

Reply:
that comment does not include `===`. behaviour equal and `===` is consistent with other yii code,